media/js/mozorg/home/home.js

```diff
@@ -2,147 +2,203 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-(function(Waypoint) {
```

Comment:
ESLint is complaining about the global variables used by the YouTube API, so you may need to add exceptions for them here at the top of this file:

```javascript
/* global YT */
/* eslint no-unused-vars: [2, { "varsIgnorePattern": "onYouTubeIframeAPIReady" }] */
```